src/smallestPrimeFactor.js

```diff
@@ -11,13 +11,45 @@
  * @param {number} num - A number
  * @returns {number} The smallest prime factor of num
  */
+function isPrime(num) {
```

Comment:
If I asked you to check whether `1547` was prime using a pen+paper and a basic calculator that could do addition, multiplication, subtraction, division, and modulus, how would you do it?

I suspect you'd do almost the inverse of what you're doing here and find the smallest proper prime factor in order to answer the question. That is, the following function would be an accurate reflection of what you as a human would do to perform `isPrime`:

```javascript
function isPrime(num) {
  if (num < 2) {
    return false;
  }

  return smallestPrimeFactor(num) === num;
}
```